test/integration/full/smoke.spec.js

```diff
@@ -0,0 +1,28 @@
+import { describe, it, expect } from '@jest/globals';
+import { configureServer } from '../../../src/server';
+import supertest from 'supertest';
+import * as cheerio from 'cheerio';
+
+describe('Smoke test', () => {
+  let server;
+  let request;
+
+  beforeEach(async () => {
+    server = await configureServer();
```

Comment:
Redundant await.

Reply:
We do need await and I just pushed NOSONAR for this line.